src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## Unreleased
 
-* Update OTel API version to `1.3.2`.
-  ([#917](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/917))
+* Update OTel API version to `1.4.0`.
+  ([#1038](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1038))
+* Added a direct dependency on System.Reflection.Emit.Lightweight which
```

Comment:
@Kielek Could you please clarify why we need to add this?

Reply:
I got my answer.

Reply:
I think it is good practice to document any new dependency.
The same was done for OTLP package https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md#140-rc4